README.md

```diff
@@ -59,15 +59,16 @@ This requires Clang and LLVM of the appropriate version, which you can
 [obtain from the repository](http://clang.llvm.org/get_started.html),
 [by download](http://llvm.org/releases/download.html).
 
-You should be able to build c2ffi with out-of-the-box clang-3.7 on your dist.  **However**, see "Notes" below for some things to watch out for.
```

Comment:
Removed version, to make the instruction more generic; we won't have to remember to update that number when a new version of LLVM is supported.